modules/oadp-deleting-cluster-resources-following-failure.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-cluster-resources-following-failure_{context}"]
+= Deleting cluster resources following a partially successful or a failed backup and restore using Data Mover
```

Comment:
Does this title mean deleting for a backup _that used_ Data Mover? If so, then I would use that wording instead of "using". "using" here sounds like you're using Data Mover to delete the cluster resources

```suggestion
= Deleting cluster resources following a partially successful or a failed backup and restore that used Data Mover
```